### DISCUSSION TITLE
How to switch to bluetooth input?

### DISCUSSION THREAD

**Discussion by Mightyflea64** [rern/rAudio#1780] [General]
Hi all,

I just installed a fresh install of rAudio-64bit-20240818.img and updated it to version 'rAudio 202409141'.
I am able to connect my Pixel 7 smartphone to my RPi4 via bluetooth but when i play music via the Tunein or Spotify app on my smartphone, the selected webradio station keeps on playing and not the music from my smartphone. My smartphone tells me it is streaming to rAudio.
How do I switch to bluetooth as input source?

**Comment by kan-ibal** [comment]
You need to have rAudio not playing before Bluetooth connecting to force sink mode.

**Reply by Mightyflea64** [reply]
Thank you for your answer, this helped me to get it working. Strangely enough when webradio plays and bluetooth plays, you can hear both streams mixed as output. I was also expecting the bluetooth symbol being displayed in the middle button.
Also there is a big difference in volume between webradio and bluetooth. Can this be adjusted in some way?

**Reply by kan-ibal** [reply]
It depends on your bluetooth output. Increase your volume at max. If it is not enough, search for a volume booster. There is an app.

**Reply by Mightyflea64** [reply]
Thank you, I will try that if necessary!
